pom.xml

```diff
@@ -422,12 +422,12 @@
                 <plugin>
                     <groupId>org.codehaus.mojo</groupId>
                     <artifactId>jaxb2-maven-plugin</artifactId>
-                    <version>2.2</version>
+                    <version>2.5.0</version>
                     <dependencies>
                         <dependency>
                             <groupId>org.jvnet.jaxb2_commons</groupId>
                             <artifactId>jaxb2-basics</artifactId>
-                            <version>0.11.0</version>
+                            <version>1.11.1</version>
```

Comment:
maybe extract this into a property? Since the version appears twice.

Reply:
The version "1.11.1" indeed appears twice, but for another artifact named "jaxb2-basics*-runtime*". Or are you referring to something else?

Reply:
If so, I can fix that in a follow-up PR.